pkgs/development/web/insomnia/designer.nix

```diff
@@ -0,0 +1,92 @@
+{ stdenv, makeWrapper, fetchurl, dpkg, alsaLib, atk, cairo, cups, dbus, expat
+, fontconfig, freetype, gdk-pixbuf, glib, gnome2, nspr, nss, gtk3, gtk2
+, at-spi2-atk, gsettings-desktop-schemas, gobject-introspection, wrapGAppsHook
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, nghttp2
+, libudev0-shim, glibc, curl, openssl, autoPatchelfHook }:
+
+let
+  runtimeLibs = stdenv.lib.makeLibraryPath [
+    curl
+    glibc
+    libudev0-shim
+    nghttp2
+    openssl
+    stdenv.cc.cc
+  ];
+in stdenv.mkDerivation rec {
+  pname = "insomnia-designer";
+  version = "2020.2.2";
+
+  src = fetchurl {
+    url =
+      "https://github.com/Kong/insomnia/releases/download/designer@${version}/Insomnia.Designer-${version}.deb";
```

Comment:
Is there a reason why you are using the debian package instead of building from source?

Reply:
I take the same approach as the currently available insomnia. I imagine is feasible to build it from source but I didn't try it and this worked well enough for me.